test/quantization/test_quantize.py

```diff
@@ -571,6 +575,115 @@ def forward(self, indices, offsets, per_sample_weights, linear_in):
         self.checkLinear(model.fc)
         self.checkDynamicQuantizedModule(quantized_model.emb, torch.nn.quantized.EmbeddingBag, torch.quint8)
 
+    @skipIfNoFBGEMM
+    def test_custom_module_class(self):
```

Comment:
Similar request as in the previous PR, lets add tests for each of the workflows (PTQ, Dynamic and QAT) and also for modules with multiple input/outputs (LSTMs).

Reply:
for dynamic/qat, will do that after the mixed mode support change.
lstm test will be done after zafar write the lstm pr. but I'll add multiple input/output support in a separate PR